numeral.js

```diff
@@ -227,6 +227,37 @@
         return Number(seconds);
     }
 
+    function formatUnits (value) {
```

Comment:
I thought it when I wrote it, but decided against it for no specific reason. Since you had the same thought, I agree.